AutomatedTests/src/main/resources/application.yml

```diff
@@ -7,3 +7,6 @@ test:
   password: ${BASIC_AUTH_PASS}
   username: ${BASIC_AUTH_USER}
   api-host: ${API_HOST}
+  api-host-secure: ${API_HOST_SECURE}
+server:

```

Comment:
please remove this part. It will break deployment

Reply:
@ekwanNTT  Done